CHANGELOG.rst

```diff
@@ -472,6 +482,11 @@ Changed
 Updates of upstream application versions
 ''''''''''''''''''''''''''''''''''''''''
 
+- In the :ref:`debops.gitlab_runner` role, the example showing how to store the
+  `gitlab_runner__token` in the inventory has been updated. The example now shows
+  how to store the token inside the Gitlab server credentials
+  under :file:`ansible/secret/credentials/`.
+
```

Comment:
I think that this should be moved higher in the file, currently it's located in the `v1.2.0` release section.